Makefile

```diff
@@ -39,6 +37,16 @@ build::
 		sed -i.bak -e "s/\$${VERSION}/$(PYPI_VERSION)/g" -e "s/\$${PLUGIN_VERSION}/$(VERSION)/g" ./bin/setup.py && \
 		cd ./bin && $(PYTHON) setup.py build sdist
 
+provider::
+	go install -ldflags "-X github.com/pulumi/pulumi-gcp/pkg/version.Version=${VERSION}" ${PROJECT}/cmd/${PROVIDER}
+
+tfgen::
+	go install -ldflags "-X github.com/pulumi/pulumi-gcp/pkg/version.Version=${VERSION}" ${PROJECT}/cmd/${TFGEN}
+
+install_plugins::
+	curl -fsSL https://get.pulumi.com | sh
```

Comment:
Should we just require that `pulumi` is on the path, rather than auto-installing from `get.pulumi.com`?

Reply:
Long term I think that's a better plan (or alternatively, we might want to make tf2pulumi learn to install providers so that having pulumi becomes unnecessary)

Reply:
I think the approach in cf4bb76 is a happy medium for now - it won't result in surprising results on developer machines, but will install in CI without making modifications to the scripts for now. Once we figure out a better story for plugins here I'll change the approach.